lectia-2.js

```diff
@@ -1,3 +1,21 @@
-// https://miorita.scoalaweb.ro/lessons/2-if
-// Scrie un program care să o ajute pe Miorița să ajungă la oricare din pășuni
-// Programul trebuie să funcționeze și cu gard, și fără gard
+reset();
+
+let free;
+free = canMove();
+
+if (free)
+{
+move();
+move();
+move();
```

Comment:
Instrucțiunile dintr-un bloc trebuie indentate. Recomand să folosești cel puțin 2 spații.